src/panels/lovelace/elements/hui-state-badge-element.js

```diff
@@ -0,0 +1,56 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../components/entity/ha-state-label-badge.js';
+
+import ElementClickMixin from '../mixins/element-click-mixin.js';
+
+/*
+ * @appliesMixin ElementClickMixin
+ */
+class HuiStateBadgeElement extends ElementClickMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          cursor: pointer; 
+        } 
+      </style>
+      <template is="dom-if" if="[[_stateObj]]">
```

Comment:
Should we also show something if state object is not found?

Reply:
I don't think we need it, and I'm not sure people would want that for an entity that is unavailable sometimes. I only did this because the `<ha-state-label-badge>` element logs console errors if stateObj doesn't exist, and it doesn't exist here until the first `_hassChanged`.

Reply:
One could consider only creating it after the first `hass` has been received 😉 Most of our normal cards are not using the Polymer template engine but instead using document.createElement.

Reply:
How should we handle a device that goes offline temporarily? Just hide it with css until we get a new stateObj?

Reply:
If a device goes offline, the state changes to unavailable. Should just work? It hardly happens that an entity is removed from the state machine.